MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+<td>Gets or sets the maximum date that users can assign to the date editor.</td>
+</tr>
+
+<tr>
+<td>{{'[MinMonth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MinMonth)'| markdownify }}</td>
+<td>Nil</td>
+<td>Gets or sets the minimum month of a date.</td>
+</tr>
+
+<tr>
+<td>{{'[MaxMonth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MaxMonth)'| markdownify }}</td>
+<td>Nil</td>
+<td>Gets or sets the maximum month of a date.</td>
+</tr>
+
+<tr>
+<td>{{'[MinYear](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MinYear)'| markdownify }}</td>
+<td>Nil</td>
+<td>Gets or sets the minimum year of a date.</td>
+</tr>
+
+<tr>
+<td>{{'[MaxYear](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MaxYear)'| markdownify }}</td>
+<td>Nil</td>
+<td>Gets or sets the maximum year of a date.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>DisplayFormat</td>
+<td>Gets or sets the attribute date display format of date editor.</td>
+</tr>
+</table>
+
+### DataFormValueConverterAttribute 
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ConverterType](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.ConverterAttribute.html#Syncfusion_XForms_DataForm_ConverterAttribute_ConverterType)'| markdownify }}</td>
+<td>ConverterType</td>
+<td>Gets the Converter type.</td>
+</tr>
+</table>
+
+## Enums
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[CommitMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.CommitMode.html)'| markdownify }}</td>
+<td>DataFormCommitMode</td>
+<td>Defines the constants for CommitMode that specifies when the value entered by user should be saved to the data object.</td>
+</tr>
+
+<tr>
+<td>{{'[LabelPosition](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelPosition.html)'| markdownify }}</td>
+<td>DataFormLabelPosition</td>
+<td>Defines the constants for label position that specifies whether to layout the label at left or top of editor.</td>
+</tr>
+
+<tr>
+<td>{{'[SuggestionMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SuggestionMode.html)'| markdownify }}</td>
+<td>DataFormTextSearchMode</td>
+<td>Specifies the type of the text search mode in ComboBox and AutoComplete.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidationMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.ValidationMode.html)'| markdownify }}</td>
+<td>DataFormValidationMode</td>
+<td>Defines the constants for ValidationMode that specifies when the user input should be validated for data object.</td>
```

Comment:
<td>Defines the constants for the ValidationMode that specifies when the user input should be validated for the data object.</td>